ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -12975,8 +12982,16 @@ void HqlCppTranslator::doBuildAggregateProcessTransform(BuildCtx & ctx, BoundRow
             }
             break;
         }
+
+        if (isAggregateOp && isDynamicOffset)
```

Comment:
Do we care that build* are called when they might only hit this condition and throw? I'm guessing this is here for code neatness rather than being checked in each aggOp case statement even though you do so in all but one for throwing HQLERR_ConditionalAggregateVarOffset?